app/src/lib/get-os.ts

```diff
@@ -83,3 +83,7 @@ export const isWindowsAndNoLongerSupportedByElectron = memoizeOne(
 export const isMacOSAndNoLongerSupportedByElectron = memoizeOne(
   () => __DARWIN__ && systemVersionLessThan('10.15')
 )
+
+export const isOSNoLongerSupportedByElectron =
+  isMacOSAndNoLongerSupportedByElectron() ||
+  isWindowsAndNoLongerSupportedByElectron()
```

Comment:
I think this should be a function (memoized or not):
```suggestion
export const isOSNoLongerSupportedByElectron = memoizeOne(
  () =>
    isMacOSAndNoLongerSupportedByElectron() ||
    isWindowsAndNoLongerSupportedByElectron()
)
```